test/sequential/rolldown.forceUpdate.test.ts

```diff
@@ -96,6 +96,28 @@ it("Validate that forceCancelRequestsFromL1 can't be called by non-sudo user", a
   });
 });
 
+it("Validate that forceCancelRequestsFromL1 don't slash user", async () => {
+  await SequencerStaking.setupASequencer(testUser, chain);
+  const stakeAmountBefore = await SequencerStaking.sequencerStake(
+    testUser.keyRingPair.address,
+    chain,
+  );
+  const { disputeEndBlockNumber } = await createAnUpdate(testUser, chain);
+  const cancel = await Rolldown.forceCancelRequestFromL1(
+    chain,
+    disputeEndBlockNumber,
+  );
+  await Sudo.asSudoFinalized(Sudo.sudo(cancel)).then(async (events) => {
+    await waitSudoOperationSuccess(events);
+  });
+  await waitBlockNumber(disputeEndBlockNumber.toString(), 25);
```

Comment:
wait a few more blocks, like +5, to avoid false positives, I remember that slashing happens some blocks after the extrinsic is submitted.

Reply:
I deleted this in the last moment because I thought you would want to reduce test time :) Added